### DISCUSSION TITLE
Side to side sway

### DISCUSSION THREAD

**Discussion by Jack-Storment** [gatsbyjs/gatsby#32256] [Help]
## Description

I was creating a client's website using Gatsby and he said that there is a "side to side sway" that occurs. Is this a Gatsby bug or is this an issue with the Theme?

### Steps to reproduce

1) Create website using Gatsby Starter Theme Kaldi. 
2) Open website on mobile browser (he uses Safari).
3) Try scrolling up and down and then to the side. Apparently if you scroll to the side, the page gets stuck.

### Expected result

Scrolling process should continue seamlessly to move about the website so clients will not exit website.

### Actual result

Upon scrolling to the side, the page gets stuck.

**Comment by Jack-Storment** [comment]
Anybody know if this is a Gatsby bug?

**Comment by Jack-Storment** [comment]
Still having an issue with this

**Comment by wardpeet** [comment]
Can you share the live site?
